sdk/monitor/Azure.Monitor.Query/tests/MetricsQueryClientTests.cs

```diff
@@ -60,5 +63,45 @@ public void ExposesPublicEndpoint()
             var client = new MetricsQueryClient(new Uri("https://management.azure.gov"), new MockCredential(), new MetricsQueryClientOptions());
             Assert.AreEqual(new Uri("https://management.azure.gov"), client.Endpoint);
         }
+
+        [Test]
+        public void CanGetMetricQueryResult()
+        {
+            MetricTimeSeriesElement metricTimeSeriesElement = new Models.MetricTimeSeriesElement();
+            Assert.IsNotNull(metricTimeSeriesElement);
+            Assert.AreEqual(0, metricTimeSeriesElement.Metadata.Count);
+            Assert.AreEqual(0, metricTimeSeriesElement.Values.Count);
+            IEnumerable<MetricTimeSeriesElement> metricTimeSeriesElements = new[] { metricTimeSeriesElement };
+
+            MetricUnit metricUnit = new MetricUnit("test");
+            Assert.IsNotNull(metricUnit);
+            Assert.AreEqual("test", metricUnit.ToString());
+
+            MetricResult metricResult = new Models.MetricResult("https://management.azure.gov", "type", new LocalizableString("name"), metricUnit, metricTimeSeriesElements);
+            Assert.IsNotNull(metricResult);
+            Assert.AreEqual(null, metricResult.Description);
+            Assert.AreEqual(null, metricResult.Error.Code);
+            Assert.AreEqual(null, metricResult.Error.Message);
+            Assert.AreEqual("https://management.azure.gov", metricResult.Id);
+            Assert.AreEqual("name", metricResult.Name);
+            Assert.AreEqual("type", metricResult.ResourceType);
+            Assert.AreEqual(1, metricResult.TimeSeries.Count);
+            Assert.AreEqual("test", metricResult.Unit.ToString());
+            IEnumerable<MetricResult> metricResults = new[] { metricResult };
+
+            MetricsQueryResult metricsQueryResult = new Models.MetricsQueryResult(null, TimeSpan.FromMinutes(3).ToString(), null, "namespace", "eastus", metricResults.ToList());
```

Comment:
Is this MetricsQueryResult constructor public?

Reply:
yes